viam-orb-slam3.go

```diff
@@ -487,34 +464,27 @@ func NewBuiltIn(ctx context.Context, deps registry.Dependencies, config config.S
 		return nil, errors.Wrap(err, "configuring camera error")
 	}
 
-	modelName := string(config.Model.Name)
-	slamLib, ok := slam.SLAMLibraries[modelName]
-	if !ok {
-		return nil, errors.Errorf("%v algorithm specified not in implemented list", modelName)
-	}
-
-	slamMode, ok := slamLib.SlamMode[svcConfig.ConfigParams["mode"]]
-	if !ok {
+	subAlgo := SubAlgo(svcConfig.ConfigParams["mode"])
+	validAlgos := []SubAlgo{Mono, Rgbd}
```

Comment:
I would put this as a const at the top.

Reply:
You say const but [] types cannot be const. Do you know the best way to do this in go? Should I just put it as a var?

Reply:
Also planning on changing the name to `supportedSubAlgos` unless you have a better suggestion

Reply:
You're right. It has to be a var! I didn't think as I wrote. `supportedSubAlgos` sounds great!